src/internal/context/form-field-context.ts

```diff
@@ -47,7 +47,11 @@ export interface FormFieldValidationControlProps extends FormFieldControlProps {
   invalid?: boolean;
 }
 
-export const FormFieldContext = createContext<FormFieldValidationControlProps>({});
+interface InternalFormFieldControlProps extends FormFieldValidationControlProps {
+  __useReactAutofocus?: boolean;
```

Comment:
One more call out, for the record. We need to find another way to pass it down, or possibly make it mandatory for consumers to use `autoFocus` on their end

Reply:
Addressed (removed) in https://github.com/cloudscape-design/components/pull/916